pages/common/git-authors.md

```diff
@@ -0,0 +1,17 @@
+# git authors
+
+> Display log or list of committers of a Git repository. If you aren't using the list command, a new file called AUTHORS will be created. Sorted by amount of commits.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-authors>.
+
+- Display log of committers of the current Git repository, reading from the AUTHORS file, opened in your default editor:
```

Comment:
```suggestion
- Display a log of committers in the default editor, reading from the `AUTHORS` file:
```

Reply:
The editor is required since the list version doesn't

Reply:
I haven't removed it 🤔

Reply:
so you haven't

Reply:
ok then commited